forge-gui/res/cardsfolder/upcoming/incessant_provocation.txt

```diff
@@ -0,0 +1,7 @@
+Name:Incessant Provocation
+ManaCost:3 R
+Types:Sorcery
+A:SP$ GainControl | ValidTgts$ Creature | TgtPrompt$ Select target creature | LoseControl$ EOT | AddKWs$ Haste | Untap$ True | SubAbility$ DBEffect | SpellDescription$ Gain control of target creature until end of turn. Untap that creature. It gains haste until end of turn. It perpetually gains "This creature attacks each combat if able."
+SVar:DBEffect:DB$ Effect | RememberObjects$ Targeted | StaticAbilities$ PerpetualAbility | Name$ Incessant Provocation's Perpetual Effect | Duration$ Permanent
+SVar:PerpetualAbility:Mode$ MustAttack | RememberObjects$ Targeted | Affected$ Creature.IsRemembered | AddHiddenKeyword$ CARDNAME attacks each combat if able. | EffectZone$ Command | Duration$ Permanent | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ This creature perpetually gains "This creature attacks each combat if able."
```

Comment:
Almost correct:
RememberObjects + Duration + AddHiddenKeyword have no meaning for this static ability

Now from rules perspective if it would just say: "That creature attacks each combat if able." then it would be correct
But it **gains** the ability which means other effects which remove all abilities could affect that

So you basically want a Continuous static which then adds the MustAttack static - look at e.g. _Absorb Energy_